src/CBT.NuGet/Tasks/NuGetRestore.cs

```diff
@@ -45,6 +45,7 @@ public class NuGetRestore : DownloadCommandBase
 
         public override bool Execute()
         {
+            EnvironmentVariables = new[] { $"CBTEnablePackageRestore=false", $"CBTModulesRestored=true" };
```

Comment:
Put a comment here explaining why this is needed and have each item on a new line so its more obvious that two vars are being set.